CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 20211209
+
+- initial ecs version: 1.57.1-1
+- initial source al1 ami: amzn-ami-minimal-hvm-2018.03.0.20211201.0-x86_64-ebs
```

Comment:
nit: "initial" should be removed from all of these

Reply:
Updated, thanks!